App.js

```diff
@@ -20,18 +14,17 @@ import { determineLanguage } from 'locale';
 import { GitPoint } from './routes';
 import { configureStore } from './root.store';
 
-const styles = StyleSheet.create({
-  logoContainer: {
-    backgroundColor: colors.white,
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-  },
-  logo: {
-    width: 100,
-    height: 100,
-  },
-});
+const Container = styled.View`
+  align-items: center;
+  background-color: ${() => colors.white}
```

Comment:
@alejandronanez I am not sure. Can it just be `background-color: 'white';` ?

Reply:
Doing it this way makes it so we can change the white color everywhere by just changing the a single value. However, why can't we just do `${colors.white}` @alejandronanez?

Reply:
Hey @chinesedfan 
- We should avoid at all costs using plain strings for colors, that's not a good practice.
- We already have a `colors` file that we should keep using.

Reply:
@alejandronanez Yes, my point is the anonymous function, as @andrewda mentioned. (You merged so quickly. 😆 )

Reply:
@andrewda you're right! There you go https://github.com/gitpoint/git-point/pull/497/files

Reply:
Oh, I put a fix for that too @chinesedfan! :)

Reply:
Understanding that this is already merged, there is no need for a function here, you can just use string interpolation

```javascript
background-color: ${colors.white};
```